default.json

```diff
@@ -124,6 +124,21 @@
       "matchPackageNames": ["eslint"],
       "allowedVersions": "^8"
     },
+    {
+      "description": "Prevent upgrade of dd-trace to v5.41.0",
+      "matchPackageNames": ["dd-trace"],
+      "allowedVersions": "!/5.41.0/"
```

Comment:
This seems okay. If you wanted to consolidate, I wonder if something like this would work?

`"allowedVersions": "!/^(5\\.41\\.0|5\\.41\\.1|5\\.42\\.0)$/"`